swap/swap.go

```diff
@@ -35,19 +35,20 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/ethereum/go-ethereum/log"
+	l "github.com/ethereum/go-ethereum/log"
```

Comment:
I think maybe swarmLog is a more appropriate name for this logger.
The loggers  are 

- swarmLog --> github.com/ethereum/go-ethereum/log
- swapLog --> github.com/ethersphere/swarm/swap/log

Reply:
since this is `go-ethereum/log`, i would just call this `log`. 

neither `swarmLog` nor `swapLog` seem like appropriate names for this import, as there is nothing related to swarm nor swap in said namespace.

as this is a naming issue, more opinions are welcome

Reply:
I switched it to log.
Naming is hard, but I think we can agree l was not a good name, log is generic but appropriate for this use.

If nobody disagrees I will keep it with log.